### PR TITLE
Initialize App_CI.yml : with @bassamEldsouki

### PR DESCRIPTION
**Pull request by ediz-genc**
Subject: Initialize App CI
type: User Story
Problem: App CI
git tags: User story, Sprint1, High

Resolves user story #20

### CODE REVIEW

**Comment by bassamEldesouki** [comment]
An App CI error because it can't find the directory we specified in the App_CI.yml file. This will be fixed after merging the App Android file to master.

File: .github/workflows/App_CI.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: |
+        cd app
```
Comment (bassamEldesouki) [inline .github/workflows/App_CI.yml]: Needs to be changed to the right directory!

**Review by bassamEldesouki — CHANGES_REQUESTED** [review]
It can't find the directory, Please write the right one!

**Comment by ediz-genc** [comment]
> It can't find the directory, Please write the right one!

I am aware : ) Waiting for the colleague to change the directory of the app so we can add the right directory.

File: .github/workflows/App_CI.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: |
+        cd tree/UI/UI/app
```
Comment (ediz-genc) [inline .github/workflows/App_CI.yml]: The directory for the app has been changed.

**Comment by platisd** [comment]
Read [how to create a commit with multiple authors](https://docs.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors). No need to tag like that on the pull request title (it doesn't provide more visibility)

File: .github/workflows/App_CI.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: |
+        cd UI
```
Comment (ediz-genc) [inline .github/workflows/App_CI.yml:25]: "Cd UI" added

**Review by bassamEldesouki — APPROVED** [review]
Fixed error  " can't find the directory of gradlew"

**Review by bassamEldesouki — APPROVED** [review]
It could be a solution!

**Comment by bassamEldesouki** [comment]
This issue has been fixed with issue #38

**Pull request closed without merging**